.github/workflows/check-release.yml

```diff
@@ -0,0 +1,35 @@
+name: Check Release
+on:
+  push:
+    branches: ['main']
+  pull_request:
+    branches: ['*']
+
+jobs:
+  check_release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install Conda environment with Micromamba
+        uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-name: cad
+          create-args: >-
+            python=3.10
+            pip
+            jupyter-releaser
```

Comment:
Do we need this step?

Reply:
Thanks! switched to the base setup action